code/game/area/areas.dm

```diff
@@ -43,11 +28,7 @@
 	/// If a room is too big it doesn't have beauty.
 	var/beauty_threshold = 150
 
-	var/requires_power = TRUE
-	/// This gets overridden to 1 for space in area/Initialize().
-	var/always_unpowered = FALSE
-
-	/// For space, the asteroid, lavaland, etc. Used with blueprints to determine if we are adding a new area (vs editing a station room)
+	/// For space, the asteroid, lavaland, etc. Used with blueprints or with weather to determine if we are adding a new area (vs editing a station room)
 	var/outdoors = FALSE
```

Comment:
This is checked on every area on a zlevel for weather, so it may be better to make this a bitflag